h/static/scripts/widget-controller.js

```diff
@@ -5,6 +5,21 @@ var angular = require('angular');
 var events = require('./events');
 var SearchClient = require('./search-client');
 
+/**
+ * Returns the group ID of the first annotation in @p results whose
```

Comment:
There isn't a formal one that I could find. Angular's docs use markdown backticks to indicate when a word refers to a parameter name. That seems like a reasonable approach.